Wikipedia/Code/UIImageView+WMFImageFetchingInternal.m

```diff
@@ -168,7 +168,8 @@ - (void)wmf_setImage:(UIImage *)image
              success:(WMFSuccessHandler)success {
     NSAssert([NSThread isMainThread], @"Interaction with a UIImageView should only happen on the main thread");
     CGRect faceBounds = [faceBoundsValue CGRectValue];
-    if (!CGRectIsEmpty(faceBounds)) {
+    CGFloat faceArea = faceBounds.size.width * faceBounds.size.height;
```

Comment:
Since we're in normalized coordinates, I'm fairly sure we can calculate the area percentage just by multiplying the bounds width and height